markdown/pxf/HivePXF.html.md.erb

```diff
@@ -172,7 +188,7 @@ Hive-plug-in-specific keywords and values used in the [CREATE EXTERNAL TABLE](..
 
 ## <a id="profile_hive"></a>Hive Profile
 
-The `Hive` profile works with any Hive file format. It can access heterogenous format data in a single table where each partition may be stored as a different file format.
+The `Hive` profile works with any Hive file format. This profile can also be used to access heterogenous format data in a single table where each partition may be stored in a different file format. Refer to the [Partition Filtering](#partitionfiltering) discussion later in this topic for additional information on partitioning and the `Hive` profile.
```

Comment:
how about "You can use this profile to access heterogenous format data in a single table where each partition may be stored in a different file format."